src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+@import "../utils/variables";
```

Comment:
I think you could import variables in the main file and it'll be available everywhere